components/automate-ui/src/app/pages/project/rules/project-rules.component.ts

```diff
@@ -73,13 +73,13 @@ export class ProjectRulesComponent implements OnInit, OnDestroy {
     private route: ActivatedRoute,
     private fb: FormBuilder) {
 
-      combineLatest(
+      combineLatest([
         this.store.select(getStatus),
         this.store.select(updateStatus),
         this.store.select(getProjectStatus)
-      ).pipe(
+      ]).pipe(
         takeUntil(this.isDestroyed),
-        map(([gStatus, uStatus, gpStatus]: string[]) => {
+        map(([gStatus, uStatus, gpStatus]) => {
```

Comment:
These are not technically strings; they are of type `EntityStatus` which is a constrained set of strings. So I could have just done this:
```
map(([gStatus, uStatus, gpStatus]: EntityStatus[]) => {
```
But, all the arguments are the same type (see my earlier note where there were different types), so the types are correctly inferred, and there is thus no need to add an explicit type here; it would be redundant.